pootle/apps/pootle_store/models.py

```diff
@@ -1832,16 +1838,14 @@ def get_common_dbids(self, opcodes, old_unitid_list, old_unitids):
 
         return update_dbids
 
-    def update(self, overwrite=False, store=None, only_newer=False, user=None,
-               submission_type=None):
+    def update(self, overwrite=False, store=None,
+               user=None, submission_type=None):
```

Comment:
My point is that this just adds another diff line when there is absolutely no need.